tensorboard/webapp/metrics/views/main_view/common_selectors.ts

```diff
@@ -26,7 +26,6 @@ import {
   getRouteKind,
   getRunsFromExperimentIds,
   getColumnHeadersForCard,
-  getCardMetadata,
```

Comment:
This isn't necessary but I'm trying to remove unused import while I find them. Now that I'm on duty these lint errors might be a good thing to try to fix.